script/cms_daily_report.rb

```diff
@@ -0,0 +1,271 @@
+# Total Enrolled Member count
+member_count = HbxEnrollment.collection.aggregate([
```

Comment:
Total members enrolled actively in a health plan regardless of auto-renewal status